tests/br_s3/run.sh

```diff
@@ -111,7 +110,6 @@ for p in $(seq 2); do
   rm -f $RESTORE_LOG
   unset BR_LOG_TO_TERM
   ( run_br restore full -s "s3://mybucket/$DB?$S3_KEY" --pd $PD_ADDR --s3.endpoint="http://$S3_ENDPOINT" \
-      --ratelimit 1 \
```

Comment:
I think maybe we need some mechanism to sync the state after removing this...

At L117-120, we sleep 3s for waiting for BR enter the phase of 'download data from external storage', then we trigger a S3 outage. Without this rate limit, the restoration may finish in 3s. And then this test case become no-meaning.

A simple mechanism to notify could be a signal file created by failpoint. (See the `s3-outage-during-writing-file`).

Reply:
chatted with Juncen, this rate limit matter to the test case, it shall be reverted.